docs/guides/configuration.md

```diff
@@ -137,14 +137,14 @@ ReactOnRails.configure do |config|
   ################################################################################
   # Server Renderer Configuration for ExecJS
   ################################################################################
-  # The default server rendering is ExecJS, probably using the mini_racer gem
+  # The default server rendering is Node.js
```

Comment:
As above, still will be ExecJS but with Node runtime

Reply:
Yes, but since it already says "for ExecJS" just 2 lines above I don't know if it needs to be said again.